mautic/mautic
# Fix and redesign sms characters count

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
We use additional fields bellow message textarea fields. Then characters count is little oversize and we made it little polish. 
Also fix one more issue



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create Sms message.
3. Save and close
3. Then try edit again
4. You should see 0 characters count
5. After this PR you should see characters count correctly
6. Also after this PR you should see characters count more decent and polish

![image](https://user-images.githubusercontent.com/462477/152818528-b822cd5c-cd15-4da6-953f-c4bc73d43a2e.png)





<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10838"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
